src/barchart/BarChart.jsx

```diff
@@ -57,9 +55,19 @@ module.exports = React.createClass({
         .rangeRoundBands([0, props.width - sideMargins], 0.1);
 
     var trans = `translate(${ margins.left },${ margins.top })`;
-    
+
     return (
-      <Chart width={props.width} height={props.height} title={props.title}>
+      <Chart
+        viewBox={props.viewBox}
+        legend={props.legend}
+        data={props.data}
+        margins={props.margins}
+        colors={props.colors}
+        colorAccessor={props.colorAccessor}
```

Comment:
Yes but not all charts, for e.g [PieChart](https://github.com/esbullington/react-d3/blob/master/src/piechart/PieChart.jsx#L49), right? Or it's been overrided?